ISISComputingGroup/EPICS-tcIoc
# Ticket 4303: Integrate CRISP jaws

See https://github.com/ISISComputingGroup/IBEX/issues/4303

This provides motor support for the PLC code running on CRISP. Documentation for how this is done is at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/tcIOC.

To test:
* Run all the tests in https://github.com/ISISComputingGroup/BeckhoffPLCCode/pull/21, see documentation for how to do this at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Beckhoff#building-and-simulating-the-code
* Ensure that disconnecting the PLC will create a comms error, this will be automated in https://github.com/ISISComputingGroup/IBEX/issues/4663
* Run a simulated beckhoff and confirm that it behaves like a normal motor
